src/main/display.cc

```diff
@@ -1239,8 +1215,9 @@ GTKChapDisp::getVerseBefore(SWModule &imodule)
 		buf = g_strdup_printf("%s<br />",
 				      (strongs_or_morph ? block_render(divBuf->str) : divBuf->str));
 		g_string_free(divBuf, TRUE);
-# endif
-#endif /* !0 */
+
+    // end grossness
+
```

Comment:
Why you do the unrelated changes?

Reply:
Not unrelated. See [this explanation](https://github.com/crosswire/xiphos/pull/913#discussion_r360250752).